src/platforms/java/common/dependencies.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Managing Dependencies
```

Comment:
Is this entirely duplicated? I can see how that's an issue without sibling inheritance (Java and Android live separately). In that case, we can do one of two things:
1. Edit this content as noted in the Android comments, and recognize we've got duplicate content
2. Put the content into a giant `include`, then set up the page, calling the `include` from both locations (a kludge, but it would ensure the content is consistent between the two platforms)

Reply:
Package names are different, and since we will remove Maven samples from Android page I think they can stay separate.